docs/examples/swift/iOS-Carthage/ThemisTest/AppDelegate.swift

```diff
@@ -23,10 +23,27 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         // We don't do UI. Please look into debug console to see the results.
         //
 
-        runExampleSecureCellSealMode()
-        runExampleSecureCellTokenProtectMode()
-        runExampleSecureCellImprint()
-
+        // generate key from pre-defined string
+        print("Using key from pre-defined string")
+        let keyFromString = self.generateMasterKey()
```

Comment:
I'd suggest to stop calling it `generateMasterKey()`, just put a constant here, and call it, like, `fixedKey`.

The `TSGenerateSymmetricKey()` is the preferred *generator* that we'd like to teach to the users.